### ISSUE TITLE
AP mode support

### ISSUE THREAD

**Issue by erwinkendo** [porjo/mt7601#23]
Good Day
Any support for AP mode with this driver?

**Comment by porjo** [comment]
Yes, I believe it is possible, however I have never used it. `iw list` will tell you what 'modes' the interface supports - you should see `AP` listed.

**Comment by erwinkendo** [comment]
Ok, that makes sense, there is even a patch that explicitly states the use of hostapd, maybe is a matter of finding the right hostapd version and configuration to support it.
Regarding issuing ```íw list```. I was under the impression that this command only works with nl80211 enabled drivers, and this one (at least with the default configuration) isn't one if them, because when I issue ```íw``` I get and empty list, but the ra0 interface is working.

For anyone interested in AP mode, [this repo](https://github.com/eywalink/mt7601u) has an AP only version of this driver, but as far as I have tested, you can't scan for surrounding APs.

**Comment by holy008** [comment]
Error (32) The connection's MAC address did not match this device when use as AP mode.
Can you help me?
Thx very much

**Comment by colinbes** [comment]
I am looking to support both AP and client mode for mt7601 chipset USB adapter. I currently have perfectly working solution for 5370 but due to chipset change in dongle I now need to support 7601. From what I can read, this requires using a mediatek (or realtek?) supplied hostapd src and building it.

Any suggestions on how to use hostapd and provide AP?
